src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const DISCOUNT_50 = 50;
+  const DISCOUNT_20 = 20;
+
+  if (days >= 7) {
```

Comment:
Good job!!!
But i would make a couple of changes. 7 and 3 we can write in constants.
![Screenshot_3](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/91065994/b91534ec-d9b1-4d9b-b2d4-80219eb4ca56)